configuration/controller/rsyslog.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+######################################################################
+# TrinityX
+# Copyright (c) 2016  ClusterVision B.V.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License (included with the sources) for more
+# details.
+######################################################################
+
+
+set -e
+
+function replace_template() {
+    [ $# -gt 3 -o $# -lt 2 ] && echo "Wrong numger of argument in replace_template." && exit 1
+    if [ $# -eq 3 ]; then
+        FROM=${1}
+        TO=${2}
+        FILE=${3}
+    fi
+    if [ $# -eq 2 ]; then
+        FROM=${1}
+        TO=${!FROM}
+        FILE=${2}
+    fi
+    sed -i -e "s/{{ ${FROM} }}/${TO//\//\\/}/g" $FILE
+}
+
+function get_partners_ip() {
+    if flag_is_unset HA; then
+        echo ''
+        return
+    fi
+    if [ "x${CTRL1_IP}" = "x" -o "x${CTRL2_IP}" = "x" ]; then
+        echo_error "Unable to determine IP of the partner in HA config. Exiting."
```

Comment:
Just use TRIX_CTRL{1,2}_IP and trust that they are set up (this happens in the critical section PS).